components/my-account/Lists/ViewLists/ViewLists.tsx

```diff
@@ -40,27 +42,46 @@ const ViewLists = (props: ListsProps) => {
     filter: publicRuntimeConfig.b2bList.filter,
   })
 
+  const [isLoading, setIsLoading] = useState<boolean>(false)
+
   // screen size declared
   const theme = useTheme()
   const mdScreen = useMediaQuery<boolean>(theme.breakpoints.up('md'))
   const { t } = useTranslation('common')
   // this will be used for creating new list
   const { user } = useAuthContext()
-  const userId = user?.userId
+  const userId = user?.id
 
   // fetching wishlist data
   const response = useGetWishlist(paginationState)
   const data = response.data as WishlistCollection
```

Comment:
Please provide a meaningful name instead of data

Reply:
done